plugins/scaffolder/src/components/fields/OwnerPicker/OwnerPicker.tsx

```diff
@@ -53,6 +54,7 @@ export const OwnerPicker = (
       defaultKind: 'Group',
       allowArbitraryValues:
         uiSchema['ui:options']?.allowArbitraryValues ?? true,
+      defaultNamespace: uiSchema['ui:options']?.defaultNamespace ?? '',
```

Comment:
You could do

```ts
const defaultNamespace = uiSchema['ui:options']?.defaultNamespace;

const ownerUiSchema = {
  ...uiSchema,
  'ui:options': {
    // ... other options
    ...( defaultNamespace !== undefined ? { defaultNamespace } : {})
  }
```

Reply:
Sorry, but you mean something like this?
```
const defaultNamespace = uiSchema['ui:options']?.defaultNamespace;

  const ownerUiSchema = {
    ...uiSchema,
    'ui:options': {
      allowedKinds: (uiSchema['ui:options']?.allowedKinds || [
        'Group',
        'User',
      ]) as string[],
      defaultKind: 'Group',
      allowArbitraryValues:
        uiSchema['ui:options']?.allowArbitraryValues ?? true,
      defaultNamespace: ( defaultNamespace !== undefined ? { defaultNamespace } : {})
    },
  };
```

that doesn't work :(

Reply:
No I think I meant precisely what I wrote, if my in-brain compiler works the way it should :)

![Screenshot 2022-09-21 at 17 05 19](https://user-images.githubusercontent.com/3097461/191540710-9e819362-abdb-4296-b793-22e2cb8e8cca.png)

Reply:
Ok this works. I have to learn about typescript a lot :-D